src/NRules/NRules.Testing/IRuleTestFixture.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using NRules.Fluent.Dsl;
+
+namespace NRules.Testing;
+
+public interface IRuleTestFixture
```

Comment:
@gritcsenko maybe better to name IRulesTestFixture given it can have multiple rules in the session?

Reply:
Renamed, thanks